### PR TITLE
Refactor Cloud to use trafficutils

### PR DESCRIPTION
**Pull request by jabb1123**
Removes any call to Trafficcloud folder

### CODE REVIEW

**Review by segerphilip — APPROVED** [review]
Looks good, test then :shipit:

File: app/handlers/uploadVideo.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import tornado.web
 from tornado.web import stream_request_body
 from tornado.httputil import parse_multipart_form_data
-from trafficcloud.api import uploadVideo 
+from traffic_cloud_utils import pm
```
Comment (dcelik) [inline app/handlers/uploadVideo.py]: I would prefer this to be

from traffic_cloud_utils.pm import create_project

**Review by dcelik — CHANGES_REQUESTED** [review]
Will need to be tested before it can be approved.

**Review by dcelik — COMMENTED** [review]
Changes address concerns, will update after testing.

**Review by dcelik — APPROVED** [review]

**Comment by dcelik** [comment]
Tested and functional. 🚢 🇮🇹

**Pull request merged**